src/components/Card/Card.test.jsx

```diff
@@ -66,13 +63,7 @@ it("should render the arrow icon", () => {
 
 it("Should render layout with children", () => {
```

Comment:
Test description doesn't match what test is doing, can you either change it and add as well snapshot test with props?